themes-default/slim/views/manage_massEdit.mako

```diff
@@ -15,14 +15,49 @@
     allowed_qualities, preferred_qualities = common.Quality.split_quality(initial_quality)
 %>
 <script type="text/javascript" src="js/quality-chooser.js?${sbPID}"></script>
-<script type="text/javascript" src="js/mass-edit.js?${sbPID}"></script>
 <script>
 let app;
 const startVue = () => {
     app = new Vue({
         el: '#vue-wrap',
         data() {
             return {};
+        },
+        mounted() {
+            function findDirIndex(which) {
+                const dirParts = which.split('_');
+                return dirParts[dirParts.length - 1];
+            }
+
+            function editRootDir(path, options) {
+                $('#new_root_dir_' + options.whichId).val(path);
+                $('#new_root_dir_' + options.whichId).change();
+            }
+
+            $('.new_root_dir').on('change', function() {
+                const curIndex = findDirIndex($(this).attr('id'));
+                $('#display_new_root_dir_' + curIndex).html('<b>' + $(this).val() + '</b>');
+            });
+
+            $('.edit_root_dir').on('click', function(event) {
+                event.preventDefault();
+                const curIndex = findDirIndex($(this).attr('id'));
+                const initialDir = $('#new_root_dir_' + curIndex).val();
+                $(this).nFileBrowser(editRootDir, {
+                    initialDir,
+                    whichId: curIndex
+                });
+            });
+
+            $('.delete_root_dir').on('click', function() {
+                const curIndex = findDirIndex($(this).attr('id'));
+                $('#new_root_dir_' + curIndex).val(null);
+                $('#display_new_root_dir_' + curIndex).html('<b>DELETED</b>');
+            });
+
+            $('#location').fileBrowser({
+                title: 'Select Show Location'
+            });
```

Comment:
From what I can see, this is what is breaking the page. **And this is not needed.**
I assume this is supposed to set the title for the folder browser when you click on Edit.
Remove this and instead add:
`title: 'Select Show Location'` to the object on lines 46-49:
```js
                $(this).nFileBrowser(editRootDir, {
                    initialDir,
					title: 'Select Show Location',
                    whichId: curIndex
                });
```
That fixed the rendering of the page for me. Folder browser tested and working.

Reply:
Tnx @sharkykh. I've committed it.